index.html

```diff
@@ -43,11 +43,108 @@ <h1 class="prof">I am Software Developer</h1>
             <a href="#" class="down-button">
                 <img src="./image/Disabled.svg" alt="">
             </a>
-        
-
+        </div>
+    </section>
 
+     <!-- content 2 -->
+    <section class="main2">
+        <div class="title">
+            <h1>My Recent Work</h1>
+            <div class="end-line"></div>
+        </div>
+        <div class="project-container">
+            <img src="./image/pic.jpeg" alt="">
+            <div class="desc">
+                <h1>Multi-Post Stories Gain+Glory</h1>
```

Comment:
- [ ] let's make sure we use `h3` tag instead of `h1` here so we follow Semantic HTML by following headers normal flow please, refer to the previous comment for more details